R/sampler.R

```diff
@@ -260,7 +260,7 @@ plot_particles <- function(particles, ylab) {
   ## Need to set plot up first to get the dates to render on the axis
   ## (matplot does not cope with this)
   dates <- as.Date(rownames(particles))
-  plot(dates, particles[, 1], type = "n", ylab = ylab, ylim = range(particles))
+  plot(dates, particles[, 1], type = "n", ylab = ylab, ylim = range(particles, na.rm = TRUE))
```

Comment:
Any other effects of potentially having `NA`s in output now? Tests look ok so I'm assuming it's fine

Reply:
yeah that was my hope too. can't see why it would impact the ylim setting as previously the trajacetories would never have NAs in as they are straight from odin